### ISSUE TITLE
lower quality image than pix2pix in lower-resolution(256*256) dataset

### ISSUE THREAD

**Issue by zoeyxu59** [NVIDIA/pix2pixHD#292]
i trained both pix2pix and pix2pixhd with my own dataset(default param and networks)，image size is 256*256,  pix2pixhd generates better image on train dataset but strange and blur result on test dataset. should I modify the G_net and D_net structure for lower resolution input image?

**Comment by songyn95** [comment]
@zoeyxu59  @aaronp24  @lukeyeager  @javl  @elezar  
I encounter the same problem. The result of my training is very good, and the test result is very blur. **There are 320 training sets in total, and the data resolution is 2560*1440.** The effects of 200, 300 and 400 iterations are tested respectively. The results are very blur. How can I solve them?  Is it because my scenario is relatively simple and there is less data?
Looking forward to your reply！！

**Comment by zoeyxu59** [comment]
@songyn95  in my case ,accuracy improved with bigger training sets (with data cleaning ,sufficient pics in each category ) .Try coarse-to-fine training, it works  when the dataset is large enough

**Comment by songyn95** [comment]
@zoeyxu59 
How much training data do you have? Can you provide your training and test commands?

**Comment by JialeXu116** [comment]
How many training pairs did you use?

**Comment by zhangxiaojuan66** [comment]
你好，邮件已收到，祝你万事如意，生活愉快！
